src/components/LocationFilter.svelte

```diff
@@ -1,27 +1,19 @@
 <script>
 import { createEventDispatcher } from 'svelte'
-import Icon from 'fa-svelte'
 import LocationInput from './LocationInput.svelte'
-import { faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons'
 
 export let title
-export let value
+export let location
 export let placeholder = 'City'
 
 const dispatch = createEventDispatcher()
 
 function onChange(event) {
-  dispatch('input', event.detail)
+  dispatch('change', event.detail)
 }
 </script>
 
   <p class="mb-1 text-muted">{ title }:</p>
   <div class="input-group mb-2">
```

Comment:
It may be worth moving the `<div class="input-group ...">` into the LocationInput.svelte file as well, since that's coupled with the classes used there for showing the map marker icon.